### PR TITLE
Persist the Multi-Valued display sub-attribute

### LINKED ISSUE #81: 'Display value' not saved
When editing a user, the field `Display value` of each type of multi-valued attribute will not be saved.

Steps to reproduce:

1. Edit a user
2. For each multi-valued attribute do:
  1. Add 1 element
  2. Type something in the field `Display value'
3. Save the user
4. Open the edit form again

Expected:

Each multi-valued attribute has its field `Display value` set with the text entered in step 2.

Actual:

The field `Display value` of each multi-valued attribute is empty.

### LINKED ISSUE #199: Display value of the multi-valued attributes is not persisted
While I wanted to fix the https://github.com/osiam/addon-administration/issues/81 bug in the administration it turns out that the display value of all multi value attributes (like email or phone number) is not persisted, because there is not even a column in the database.

### PR DESCRIPTION
**Pull request by dacrome**
Resolves #199 and closes osiam/addon-administration#81

### CODE REVIEW

**Comment by tkrille** [comment]
This will be done after OSIAM 3.0

**Comment by dacrome** [comment]
Why not with 3.0?

**Comment by tkrille** [comment]
Mostly, because yesterday was the last day to add new features or changes. Of course, that's negotiable. What do you think @wallner?

File: src/main/java/db/migration/V2__migrations_from_2_5.java [hunk 1]
```diff
@@ -47,6 +47,49 @@ public Integer getChecksum() {
     public void migrate(Connection connection) throws Exception {
         this.connection = connection;
         removeClient("auth-server");
+        addDisplayToMultiValuedAttributes();
+    }
+
+    private void addDisplayToMultiValuedAttributes() throws SQLException {
```
Comment (tkrille) [inline src/main/java/db/migration/V2__migrations_from_2_5.java:53]: Maybe this should be done with an SQL-based migration?

**Comment by tkrille** [comment]
Alright, I didn't notice, that you already created an issue for that more than a week ago. Rescheduled for 3.0.

File: src/main/java/db/migration/V2__migrations_from_2_5.java (hunk 1 above)
Comment (dacrome) [inline src/main/java/db/migration/V2__migrations_from_2_5.java:53]: But then I have to do it for every database.

Comment (tkrille) [inline src/main/java/db/migration/V2__migrations_from_2_5.java:53]: Alright. Maybe we should migrate to a Spring JDBC migration, to make this less complex, but that should be done in an upcoming PR.

**Comment by tkrille** [comment]
LGTM

**Pull request merged**